noxfile.py

```diff
@@ -1,10 +1,11 @@
 import tempfile
 
 import nox
+from packaging.version import parse as parse_version, Version
 
 import os
 import time
-from typing import List, Union
+from typing import List, Union, Dict
 
 
 def python_versions() -> Union[List[str], bool]:
```

Comment:
don't you prefer to make this function to accept an argument like 'version_level'?

Reply:
What do you mean by 'version_level'?

Reply:
Added